### ISSUE TITLE
Unable to import Vendor information from SBOM because Vendor in SW360 is an object

### ISSUE THREAD

**Issue by afsahsyeda** [eclipse-sw360/sw360#2394]
[//]: # (Copyright Siemens AG, 2021. Part of the SW360 Portal Project)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

### Description

Unable to import Vendor information from SBOM because Vendor in SW360 is an object that requires Name, short name and URL as mandatory fields whereas only "Name" is available in the SBOM for vendor information.

**Comment by Suvrat1629** [comment]
Hey @afsahsyeda i was taking a look at this issue and I think changing the shortName and url fields from `required` to `optional` in the vendor.thrift file would help. But I am confused as to how I would make this reflect in the rest of the codebase? Do i add the `VendorService.Iface` in the SbomImporter.java to try to reflect the changes of the vendor.thrift file changes? Could you guide me a bit.
Thank you.

**Comment by afsahsyeda** [comment]
@Suvrat1629 Yes we would make the fields optional in vendor.thrift but we are not sure yet if we could completely rely on the vendor information from SBOM because usually the vendor info is missing from components. 
Can you look into this issue instead: https://github.com/eclipse-sw360/sw360/issues/2379
It is important and also related to SBOM topic.
